src/AuditTrailModule/javasource/test_crm/tests/expected/ExpectedLog.java

```diff
@@ -0,0 +1,93 @@
+package test_crm.tests.expected;
+
+import static org.junit.Assert.*;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import com.mendix.core.CoreException;
+import com.mendix.systemwideinterfaces.core.IMendixIdentifier;
+
+import audittrail.proxies.Log;
+import audittrail.proxies.TypeOfLog;
+import test_crm.tests.actual.ActualLog;
+import test_crm.tests.actual.ActualLogLine;
+
+public class ExpectedLog {
+	final Integer numberOfChangedMembers;
+	final TypeOfLog typeOfLog;
```

Comment:
I documented `typeOfLog`, but left as is, since it's at least consistent with the enum.

`objectType` was just as confusing as the `LogObject` attribute, imo. I changed to `entityTypeName` and documented the field to explain this difference.